{"name": "test_recuperar_a_lista_de_livros", "status": "broken", "statusDetails": {"message": "pytest_bdd.exceptions.StepDefinitionNotFoundError: Step definition is not found: Given \"que estou autenticado na API\". Line 4 in scenario \"Recuperar a lista de livros\" in the feature \"/Users/daniloamedeiros/Desktop/project-challenge-qa-acc/features/api_books.feature\"", "trace": "E   pytest_bdd.exceptions.StepDefinitionNotFoundError: Step definition is not found: Given \"que estou autenticado na API\". Line 4 in scenario \"Recuperar a lista de livros\" in the feature \"/Users/daniloamedeiros/Desktop/project-challenge-qa-acc/features/api_books.feature\"\nAll traceback entries are hidden. Pass `--full-trace` to see hidden and internal frames."}, "description": "/Users/daniloamedeiros/Desktop/project-challenge-qa-acc/features/api_books.feature: Recuperar a lista de livros", "attachments": [{"name": "log", "source": "60d55297-9b61-416d-a43a-a653854f3bf3-attachment.txt", "type": "text/plain"}], "start": 1739370990774, "stop": 1739370990774, "uuid": "ce3bcbb4-7d3b-48b2-a0bb-af8ff4bb61d2", "historyId": "65d197b98b23e192ecc0cd4d2d9fbe6d", "testCaseId": "65d197b98b23e192ecc0cd4d2d9fbe6d", "fullName": "tests.api.test_books#test_recuperar_a_lista_de_livros", "labels": [{"name": "parentSuite", "value": "tests.api"}, {"name": "suite", "value": "test_books"}, {"name": "host", "value": "MacBook-Pro-de-Danilo.local"}, {"name": "thread", "value": "51247-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.api.test_books"}]}